looker_deployer/commands/deploy_content.py

```diff
@@ -50,6 +50,10 @@ def create_or_return_space(space_name, parent_id, sdk):
 
     try:
         target_id = get_space_ids_from_name(space_name, parent_id, sdk)
+        if len(target_id) == 0 and "/" in space_name:
+            # If the folder name contains slashes then also check if it was previously imported with
+            # the slashes replaced with division slashes (Unicode character 2215) prior to PR #152.
```

Comment:
Since a new PR was created the PR reference in this comment needs to be updated, but I don't appear to have permission to do so on this local branch.
```suggestion
            # the slashes replaced with division slashes (Unicode character 2215) prior to PR #153.
```